src/NewTools-Debugger/StDebugger.class.st

```diff
@@ -750,6 +666,15 @@ StDebugger >> forceSessionUpdate [
 		updateExtensionsFromSession: self session
 ]
 
+{ #category : 'actions' }
+StDebugger >> implementMissingMethod [
+
+	self debuggerActionModel implementMissingMethod.
+	self selectTopContext.
+	self flag: 'why do we need to give code the keyboard focus?'
+	"code takeKeyboardFocus"
```

Comment:
The line of code has been added in 2020 to fix this issue: https://github.com/pharo-spec/NewTools/issues/123

We have to check if it is still necessary

Reply:
I would leave it for now